docs/it-manual/sre-playbook/troubleshooting-production.md

```diff
@@ -1,11 +1,68 @@
 # Troubleshooting Production
 
-Is something broken in production? Did you get paged? Here are the most common issues.
+## AWS
 
-## Servers are down / not responding
+### Outage, site is down
 
-## User is unable to authenticate
+#### DNS
```

Comment:
suggestion (non-blocking): in the rich diff, `DNS` (and other headers with `####`) are visually the same size as `Symptoms`, `Diagnose`, and `Resolution` which makes it hard to visually parse the section separation. I suggest making `DNS` bigger or underlining it to make it clear that that is the header for this section.

Reply:
i don't think i'm seeing that change? they still look the same size to me: 
<img width="176" alt="image" src="https://github.com/civiform/docs/assets/24319951/5e73211a-f23e-4210-8ee4-169d9440efb7">

Reply:
Hadn't pushed the changes yet, been sneaking edits to this during meetings all day 😆 .

Just pushed and re-requested review